telegram/_dice.py

```diff
@@ -93,3 +106,45 @@ def __init__(self, value: int, emoji: str, *, api_kwargs: JSONDict = None):
     """
```

Comment:
I didn't add any `.. versionadded` to class constants that I didn't create, though I think this needs to be done. I also think that the version marking in the line above this one might be wrong: a bowling emoji was added in 13.4, but the constant itself is being introduced in 20.0. So maybe all constants referring to  `DiceEmoji` should also get the `20.0` marking.